logger.go

```diff
@@ -336,3 +336,9 @@ func (logger *Logger) AddHook(hook Hook) {
 	defer logger.mu.Unlock()
 	logger.Hooks.Add(hook)
 }
+
+func (logger *Logger) ReplaceHooks(hooks LevelHooks) {
```

Comment:
Can we have the former hooks returned from this method ? It could practical in order to restore the former hooks.